debian/control

```diff
@@ -0,0 +1,29 @@
+Source: key-mapper
+Section: misc
+Priority: optional
+Maintainer: Sezanzeb <proxima@sezanzeb.de>
+Standards-Version: 4.5.0
```

Comment:
The current version is 4.5.1:

```suggestion
Standards-Version: 4.5.1
```

Reply:
I ran it on ubuntu 20.04 and got a lintian warning: `key-mapper source: newer-standards-version 4.5.1 (current is 4.5.0)` so I downgraded.

Reply:
The purpose of the field is to track the version of the standards which it adheres to; your packaging is compliant with 4.5.1, so it should be declared as such, even if older versions of lintian don’t know about it.